src/_assets/js/main.js

```diff
@@ -22,12 +22,7 @@ $(function () {
   setupClipboardJS();
 
   // New (dash) tabs
-  setupTabs($('#editor-setup'), 'io.flutter.tool-id');
-  setupTabs($('#engine-language'), 'io.flutter.tool-id');
-  setupTabs($('#vc-language'), 'io.flutter.tool-id');
-  setupTabs($('#no-engine-vc-language'), 'io.flutter.tool-id');
-  setupTabs($('#entrypoint-library-language'), 'io.flutter.tool-id');
-  setupTabs($('#initial-route-language'), 'io.flutter.tool-id');
+  setupTabs($('.nav-tabs'), 'io.flutter.tool-id');
```

Comment:
rather than repurposing one of these existing bootstrap classes, let's just make a new one so we can make sure everything's clean and we don't affect other things

Reply:
Done

Reply:
Seems like you changed the various tabs themselves to have sample-code-tabs but didn't change the scan here. Can you also change the iOS page's tabs as well to use the new class?

Reply:
Here is a PR that introduces a new Liquid tag and adds that selector change that I missed:
https://github.com/flutter/website/pull/3350